src/app/article/article.component.scss

```diff
@@ -140,20 +147,24 @@
 
 /* For tablets: */
 @media only screen and (min-width: 600px) and (max-width: 768px) {
-	#single-article-container {
+	:host /deep/ #single-article-container {
```

Comment:
Very well observed, :host handle can be useful at some times, but in this case it's more efficient to **just use #single-article-container** as the rules of each scss file is contained in the component (if no special override directives are present).